styles/core/utilities.css

```diff
@@ -12,6 +12,11 @@
   justify-content: center;
 }
 
+.flex-wrap {
+  flex-wrap: wrap;
+  row-gap: 10px;
+}
+
 .max-width-height {
   width: 100vh;
```

Comment:
Typo: Maybe width should be 100vw as it is the max-width

Reply:
Thanks for pointing that out

Reply:
fixed - 5653b94ff86ab74afb718f960495227fa419cb64

Reply:
Okay to merge